apps/microsoft-teams/app-actions/src/actions/send-test.ts

```diff
@@ -1,16 +1,88 @@
 import { AppActionCallContext } from '@contentful/node-apps-toolkit';
 import { AppActionCallResponse } from '../types';
+import { fetchTenantId } from '../utils';
+import helpers from '../helpers';
 
 interface AppActionCallParameters {
-  channel: string;
+  channelId: string;
+  teamId: string;
+  contentTypeId: string;
+  spaceName: string;
+}
+
+export interface BotServiceResponse {
```

Comment:
We have some types that are going to work a lot better for what we want to do here 😄 .

See https://github.com/contentful/apps/blob/master/apps/ai-image-generator/backend/src/types.ts#L32-L48

You can see an example of how we type the return of our handler here:

https://github.com/contentful/apps/blob/master/apps/ai-image-generator/backend/src/actions/aiig-generate-image.ts#L18

This type is going to work better for disambiguating between success and error results.

We really need a backend toolkit (or "hosted app actions toolkit"? 🤔 ) where we can stick these common types and utilties instead of copying and pasting. 😢

Reply:
Awesome, thank you! Agreed on adding this to backend toolkit that has common types and helpers for HAA and lambdas!